cmd/fluxmon/queries.yaml

```diff
@@ -0,0 +1,11 @@
+queries:
```

Comment:
One thing I noticed is you can only select a single number for your gauge. We might, for example, want to have per-instance gauges for job queue length, or per queue (~job type). But this is a great start, and we can do that later since this is already useful.